vignettes/sensitivity-analysis.Rmd

```diff
@@ -0,0 +1,118 @@
+---
+title: "Sensitivity analysis"
+output: rmarkdown::html_vignette
+#output: pdf_document
+vignette: >
+  %\VignetteIndexEntry{Sensitivity analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The models built with OSPS depend on a lot of input parameters which are based on literature values, measurements, databases, assumptions. For a given set of input parameters a number of output curves is computed in a simulation. To assess which input parameters have most impact on the output curves, a sensitivity analysis of the simulaiton can be performed. For more information about theoretical background, see [OSPS documentation](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/sensitivity-analysis).
+
+In brief, the values of the chosen parameters are changed by a certain percentage and the impact of these changes on PK parameters of model outputs is assessed.
+
+## Performing a sensitivity analysis
+The first step of performing a sensitivity analysis is creating an object of the class `SensitivityAnalysis`. At this step, the user can define which parameters should be considered for the sensitivity analysis, in which range the values are variated, and how may steps are performed in one direction (plus and minus). If no parameters are specified, all constant parameters of the simulation will be variated.
+
```

Comment:
If no parameters are specified, all constant **and suitable for sensitivity calculations** parameters of the simulation will be variated.

"suitable for sensitivity calculations" means: 
- is not zero
- is **effectively** used in simulation (btw: is this parameter property accessible from R and if yes: is it also explained somewhere?)
- is non-categorical 
- is not marked with "CanBeVaried=false" in the simulation
- is not marked as read-only
- is not a table
@msevestre Anything else?
The list of all such parameters can be get from the simulation by `potentialVariableParameterPathsFor(simulation)`.